netmaster/k8snetwork/networkpolicy.go

```diff
@@ -47,311 +80,491 @@ func (k8sNet *k8sContext) handleK8sEvents() {
 	}
 }
 
-func (k8sNet *k8sContext) createNetwork(nwName string) error {
-	npLog.Infof("create network %s", nwName)
+func (k8sNet *k8sContext) createEpg(tenantName string, nwName, epgName string) error {
+	policyName := k8sutils.EpgNameToPolicy(epgName)
```

Comment:
k8s policies will end up creating EPGs. We should make sure these are not deleted if the user also creates policies in this EPG